packages/components/src/tree-grid/types.ts

```diff
@@ -0,0 +1,116 @@
+export type TreeGridRowProps = {
+	/**
+	 * The children to be rendered in the row.
+	 */
+	children: React.ReactNode;
+	/**
+	 * An integer value designating the level in the hierarchical tree structure.
+	 * Counting starts at 1. A value of `1` indicates the root level of the structure.
+	 */
+	level: React.AriaAttributes[ 'aria-level' ];
+	/**
+	 * An integer value that represents the position in the set.
+	 * A set is the count of elements at a specific level. Counting starts at 1.
+	 */
+	positionInSet: React.AriaAttributes[ 'aria-posinset' ];
+	/**
+	 * An integer value that represents the total number of items in the set,
+	 * at this specific level of the hierarchy.
+	 */
+	setSize: React.AriaAttributes[ 'aria-setsize' ];
+	/**
+	 * An optional value that designates whether a row is expanded or collapsed.
+	 * Currently this value only sets the correct aria-expanded property on a row,
+	 * it has no other built-in behavior.
+	 */
+	isExpanded?: React.AriaAttributes[ 'aria-expanded' ];
```

Comment:
The `Booleanish` type is fun! Although it's a slightly larger set than what the component docs originally advertised (`boolean`) — do you see any issues with widening this prop's type?

Reply:
That's a very good question. I'm going to keep it to a simple `boolean`, since the prop name already implies that it's a simple boolean (vs. a standard prop name like `aria-expanded`), and we _may_ want to do something more with the prop value in the future. Good to keep our options open.